labs/1.md

```diff
@@ -135,29 +135,20 @@ verify your conjectures.
 9. Give an example of (a) open-source software and (b) free, but closed-source
    software that you use.
 
-## Part 3: Obtaining your own VM
+## Part 3: Obtaining your VM
 Using the public OCF login server will allow you to do basic things like the lab above, but if you want root permission (which lets you do basically whatever you want), you'll need your own use/destroy!
 
-Here's the problem statement:
+We will be providing a virtual machine (VM) to all DeCal students with a Berkeley CalNet account. Check your @berekely.edu email!
```

Comment:
```suggestion
We will be providing a virtual machine (VM) to all DeCal students with a Berkeley CalNet account. Check your @berkeley.edu email!
```

Reply:
i was planning on sending them out on monday, idk if we wanted to mention that in case people read it early